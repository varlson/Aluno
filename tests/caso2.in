M
100
Bianca direito 6.3
Isabelly computacao 7.6
DaviLucas nutricao 6.0
MariaHelena direito 0.7
MariaEduarda direito 6.2
AnaSophia nutricao 0.5
Oliver direito 9.7
ArthurHenrique nutricao 7.9
Bryan computacao 1.8
Samuel nutricao 7.9
Catarina direito 5.0
Mariana direito 7.9
Rodrigo computacao 7.5
Lavinia nutricao 1.7
Nicolas nutricao 9.0
Mirella nutricao 6.2
Luan nutricao 8.2
MariaVitoria computacao 3.5
Gustavo direito 0.5
Antonia nutricao 2.8
Mirella nutricao 0.3
Ana direito 1.4
Liz direito 6.7
MariaJulia nutricao 7.0
Ryan direito 7.0
Enrico nutricao 5.5
Pietro direito 4.3
Arthur computacao 6.2
Gabrielly nutricao 7.1
Felipe direito 6.1
MariaIsis computacao 1.5
Alice computacao 4.9
Louise computacao 9.3
Esther computacao 7.8
AnaLiz direito 4.9
Francisco direito 7.8
Gael computacao 5.7
Aurora direito 8.1
Joao direito 9.6
Matheus nutricao 7.1
Victor direito 4.8
AnaLaura nutricao 3.1
DaviMiguel computacao 2.6
Maya computacao 5.7
Otavio computacao 8.5
Isadora nutricao 9.5
Cecilia computacao 5.5
Amanda nutricao 9.8
Heitor nutricao 5.9
MariaFernanda computacao 7.1
Levi direito 0.5
Malu nutricao 4.7
Cecilia direito 5.2
LuizFelipe direito 2.8
Gabrielly direito 4.1
DaviLucas nutricao 0.7
Thomas computacao 3.8
Rafael direito 8.1
Sophia direito 5.7
AnaLiz direito 7.6
Mariah computacao 4.1
Lara computacao 2.1
Davi computacao 6.9
Melissa nutricao 2.0
AnaSophia computacao 1.6
Oliver computacao 5.6
Jose nutricao 9.0
Emanuel direito 4.4
Rafaela direito 4.7
MariaHelena direito 6.2
Lorena computacao 1.2
Ana computacao 9.6
Joana nutricao 4.9
Mariah direito 6.9
Joao direito 7.8
MariaFernanda direito 3.9
Julia direito 5.2
Isis direito 7.9
Liz computacao 4.3
Luan nutricao 9.2
Jose direito 6.9
Andre computacao 9.7
MariaHelena computacao 8.7
Davi nutricao 9.9
Guilherme direito 2.9
MariaClara nutricao 1.7
PedroHenrique nutricao 6.2
Vicente computacao 0.2
Luna direito 7.7
Rafael direito 3.8
Benjamin nutricao 6.3
Henrique direito 8.6
Stella computacao 4.2
Esther computacao 0.9
Benjamin computacao 8.7
DaviLucas nutricao 5.5
Vinicius direito 6.3
Arthur nutricao 7.7
Luna nutricao 0.4
Luiza computacao 0.7