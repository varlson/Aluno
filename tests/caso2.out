A media no curso de computacao eh 5.1
A media no curso de direito eh 5.6
A media no curso de nutricao eh 5.6
